pkgs/applications/radio/cqrlog/default.nix

```
@@ -0,0 +1,100 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fpc
+, lazarus
+, atk
+, cairo
+, gdk-pixbuf
+, glib
+, gtk2-x11
+, libX11
+, pango
+
+, hamlib
+, mysql57
+, tqsl
+, xdg_utils
```

`xdg_utils` is an alias to `xdg-utils`.

```suggestion
, xdg-utils
```

---

Fixed